polypie.py

```diff
@@ -13,12 +14,10 @@
 
 
 class PolypieException(Exception):
-
```

Comment:
I prefer one blank line below `class` keyword regardless of the presence a docstring.